ios/browser/api/sync/brave_sync_api.h

```diff
@@ -44,11 +71,15 @@ OBJC_EXPORT BraveSyncAPIWordsValidationStatus const
     BraveSyncAPIWordsValidationStatusWrongWordsNumber;
 
 OBJC_EXPORT
+
```

Comment:
it seems this empty line shouldn't be here

Reply:
Removed at https://github.com/brave/brave-core/pull/17369#discussion_r1118899590